compiler/parser/parser.peg

```diff
@@ -1211,6 +1211,7 @@ IdentifierNames
   = first:IdentifierName rest:(__ "," __ id:IdentifierName { RETURN(id) })* {
     RETURN(PREPEND(first, rest))
   }
+  / __ { RETURN(ARRAY()) }
```

Comment:
s/IdentifierNames/IdentifierNames?/ in the FuncDecl rule would be clearer.

Reply:
Yeah